examples/compile-only-dependencies/coollib/MVN-INF/BUILD.pom

```diff
@@ -0,0 +1,11 @@
+maven_artifact(
+    group_id = "com.pomgen.depman.example",
+    artifact_id = "coollib",
+    version = "3.0.0-SNAPSHOT",
+    pom_generation_mode = "dynamic",
+    generate_dependency_management_pom = True,
```

Comment:
Remove this attribute, it isn't needed for this example.  Also adjust group_id to match the example name? (so it looks less copy/pasted?)